helper/pluginutil/runner.go

```diff
@@ -136,7 +136,7 @@ type APIClientMeta struct {
 }
 
 func (f *APIClientMeta) FlagSet() *flag.FlagSet {
-	fs := flag.NewFlagSet("vault plugin settings", flag.ContinueOnError)
+	fs := flag.NewFlagSet("vault plugin settings", flag.ExitOnError)
```

Comment:
Let's leave this as ContinueOnError so that plugin authors can still define their own flags